src/lib/button/_button-base.scss

```diff
@@ -62,14 +62,30 @@ $mat-mini-fab-padding: 8px !default;
   }
 }
 
-// Applies styles to buttons with backgrounds: raised, fab, and mini-fab
-%mat-raised-button {
+// Applies styles to buttons with stroke styling
+%mat-stroked-button {
```

Comment:
Why make stroked and flat sass placeholders? The raised button was a placeholder because the styles applied for both regular raised for fabs.